aljuarismi/datasets.py

```diff
@@ -31,26 +31,48 @@ def ask_for_dataset_path():
     return path
 
 
+def ask_for_dataset_extension():
+    """
+    Ask for the dataset extension file
+    :return: The extension file
+    """
+    print('What is his extension (txt,csv,...)?')
```

Comment:
What is the file extension (or its extension) (txt, csv)?

There are only two options,no need to use points